### PR TITLE
Ms edge cases

### PR DESCRIPTION
**Pull request by meghanstovall**
Added to our test file for edge cases. 
For the first test, I added another user to make sure it was only displaying the first users' repos.
Added another test to make sure the GitHub repo section wouldn't show if the user didn't have a token.

### CODE REVIEW

**Comment by cgaddis36** [comment]
I thought these builds finishing last night?

File: spec/features/user/user_can_see_their_repos_spec.rb [hunk 1]
```diff
@@ -2,19 +2,43 @@
...
+    user1_repos = 5.times.map { Repo.new(name: "User 1 Repositories", html_url: "www.github.com") }
+    user2_repos = 5.times.map { Repo.new(name: "User 2 Repositories", html_url: "www.github.com") }
+
+    allow(user1).to receive(:repos) {user1_repos}
+    allow(user2).to receive(:repos) {user2_repos}
+
+    visit '/dashboard'
+    expect(page).to have_link('User 1 Repositories', count: 5)
+    expect(page).to_not have_link('User 2 Repositories')
+  end
```
Comment (cgaddis36) [inline spec/features/user/user_can_see_their_repos_spec.rb]: nice edge case here

File: spec/features/user/user_can_see_their_repos_spec.rb [hunk 2]
```diff
@@ -2,19 +2,43 @@
...
+                      last_name: "user1",
+                      password: "hamburger1",
+                      role: 0)
+
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user1)
+
     visit '/dashboard'
-    expect(page).to have_link('User Repositories', href: 'www.github.com', count: 5)
+    expect(page).to_not have_content("Github Repositories")
   end
```
Comment (cgaddis36) [inline spec/features/user/user_can_see_their_repos_spec.rb]: also like the test for no tokens here
